### PR TITLE
Creating canvas

### PR DESCRIPTION
**Pull request by chompasina**
Separates out World and Tray modules
Creates event listeners for arrow keys
Renders world with tray that moves

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+};
+
+document.addEventListener('keydown', function(event){
+  if (event.keyCode === 39) { world.rightArrow();}
+  if (event.keyCode === 37) { world.leftArrow();}
+});
+
+requestAnimationFrame(function gameLoop(){
+  renderWorld();
+  // context.clearRect(0, 0, canvas.width, canvas.height);
```
Comment (karinamzalez) [inline lib/index.js]: do we want to keep comments out of production?

File: lib/tray.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+function Tray(settings){
+  settings = settings || {};
+  this.width = settings.width || 100;
+  this.height = settings.height || 10;
+  this.x = typeof settings.x === 'undefined';
+  this.y = settings.y || 470;
+  // this.world = settings.world;
```
Comment (karinamzalez) [inline lib/tray.js]: one last one :)

**Comment by karinamzalez** [comment]
👍

**Comment by deborahleehamel** [comment]
👍 We have a canvas and a tray that can move left and right. Good first sketch.

**Pull request merged**